docs/source/development.rst

```diff
@@ -113,6 +113,33 @@ We have adopted the `Google style guide <https://google.github.io/styleguide/she
 
 All new code after 2022 Sep 1 will be required to meet these standards. We will slowly be updating existing scripts to comply with the standards. We are also in the process of adding GitHub actions to automatically lint code submitted for PRs.
 
+------------------
+Using PyCodeStyle
+------------------
+
+To ensure your Python code meets PEP-8 standards, you can use the ``pycodestyle`` tool to check for compliance before submitting your code. The repository contains a ``.pycodestyle`` configuration file with project-specific settings.
+
+To check your code:
+
+::
+
+    pycodestyle -v --config ./.pycodestyle ./
+
+This command will recursively check all Python files in the current directory against our style configuration and provide verbose output about any issues found. You should resolve these issues before submitting a pull request.
+
+For specific files or directories:
+
+::
+
+    pycodestyle -v --config ./.pycodestyle path/to/specific/file.py
+    pycodestyle -v --config ./.pycodestyle path/to/directory/
+
+You can install PyCodeStyle using pip:
+
+::
+
+    pip install pycodestyle
+
```

Comment:
The `py-pycodestyle` module is also available in spack-stack 1.6.0+.  Loading the `module_gw_setup.<machine>` module followed by `module load py-pycodestyle` will add it to your environment without having to manage an additional Python environment.

Reply:
removed